gitless/cli/gl_remote.py

```diff
@@ -24,6 +24,9 @@ def parser(subparsers, _):
   remote_parser.add_argument(
       '-d', '--delete', nargs='+', help='delete remote(es)', dest='delete_r',
       metavar='remote')
+  remote_parser.add_argument(
+      '-rn', '--rename', nargs='+',
+      help='rename the specified remote', dest='rename_r')
```

Comment:
this looks good. Perhaps mention the fact that you expect two args (current remote name and new name)